src/lib/Tisch/Internal/Table.hs

```diff
@@ -221,54 +247,58 @@ instance Aeson.ToJSON a => Aeson.ToJSON (WDef a) where
 -- /Notice that 'Col' is very different from 'Kol' and 'Koln': 'Kol' and 'Koln'/
 -- /are used at runtime for manipulating values stored in columns, 'Col' is used/
 -- /to describe the properties of a column at compile time./
-data Column name wcap rcap pgType hsType
-   = Column name wcap rcap pgType hsType
+--
+-- Is it also used at the term level when generating CREATE statements for
+-- tables.
+data Column name wcap rcap unique pgType hsType
```

Comment:
Having this `Unique` here is not sufficient. Unique constraints may involve more than one column, so the right place to put this information is in some accompanying type family mentioning some additional properties of the table, one of which would be uniqueness properties.